format_and_overlap_determination.md

```diff
@@ -0,0 +1,118 @@
+# Description of Format and Overlap Determinations #
+This attempts to describe what this system actually does. This does not attempt to exhaustively describe all corner cases or emergent properties.
+
+## Definitions ##
+**Format**
+: mono (spm), multi (mpm), serial (ser)
+
+**Cluster**
+: Collection of HT Items, Holdings, and Concordance pairs united by a list of OCNs determined to be equivalent.
+
+**Bib Record**
+: A preferred record from the Zephir clustering system containing one or more HT Items. Has a single Hathitrust record ID.
+
+**HT Item**
+: Volume ID and associated metadata taken from the Hathifiles. 
+
+**Print Serials List**
+: List of Record Ids determined to be serials by some unknown UMich process. List is limited to those held by UMich.
+
+**Large Clusters**
+: Clusters large enough that they create implementation problems. Identified by an OCN, there are currently 11 on the list.
+
+**Billing Entity**
+: The `billing_entity` field taken from the `ht_collections` table associated with a particular HT Item through the collection code. Used for cost allocation when holdings are incomplete or no clustering can be performed. Also has subtle and likely inconsequential implications for items contributed by UCM and KEIO.
+
+**n_enum**
+: A normalized enumeration. Description fields are processed for enumeration and chronology. Only enumeration ends up being used. 
+
+
+## Item Format ##
+
```

Comment:
Is it worth saying explicitly here why item format matters?  Same for Cluster Format below.

Reply:
Done

Reply:
Is "calculate" really right here?  Like, is it math?  Or would something like "assign" capture what happens?  Sorry to nitpick.

Reply:
According to wikipedia, "To calculate means to determine mathematically in the case of a number or amount, or in the case of an abstract problem to deduce the answer using logic, reason or common sense." In this case it also matches the terminology used in the code.

Reply:
okie dokie